packages/opentelemetry/lib/index.js

```diff
@@ -150,6 +164,39 @@ function setupOpenTelemetry({ authorizationHeader, tracing } = {}) {
 		openTelemetryConfig.spanProcessor = new NoopSpanProcessor();
 	}
 
+	if (metrics?.endpoint) {
+		const headers = {
+			'user-agent': METRIC_USER_AGENT
+		};
+		if (authorizationHeader) {
+			headers.authorization = authorizationHeader;
+		}
+		const exporter = new OTLPMetricExporter({
+			url: metrics?.endpoint,
+			headers
+		});
+		openTelemetryConfig.metricReader = new PeriodicExportingMetricReader({
+			exporter,
+			exportIntervalMillis: 1000 // Set as 1000ms for development and demostration, need to set it higher for prod
```

Comment:
Leave this to the default value when you move forwards with this. Unlike Graphite we will crack down on teams shipping metrics more frequently than the default 60 seconds.

#### References

* https://github.com/open-telemetry/opentelemetry-js/blob/82b7526b028a34a23936016768f37df05effcd59/packages/sdk-metrics/src/export/PeriodicExportingMetricReader.ts#L97